src/client/utils/url-resolver.js

```diff
@@ -0,0 +1,62 @@
+import nativeMethods from '../sandbox/native-methods';
+import { ensureTrailingSlash } from '../../utils/url';
+
+const DOCUMENT_URL_RESOLVER = 'hammerhead|document-url-resolver';
+
+var urlResolver = {
```

Comment:
But it's done wrong here anyway. Just `export default {}`, no need for explicit `var` here. Use `this` to refer object fields